extensions/default/src/DicomWebDataSource/index.js

```diff
@@ -212,13 +234,14 @@ function createDicomWebApi(dicomWebConfig, userAuthenticationService) {
 
     store: {
       dicom: async (dataset, request) => {
-        wadoDicomWebClient.headers = getAuthrorizationHeader();
+        clientManager.setAuthorizationHeadersForWADO();
```

Comment:
similar to the my other comment above, move the setHeader down, similar to what you have for others